api/app/src/command/webhook/webhook.ts

```diff
@@ -120,6 +121,16 @@ export const processData = async <T extends MetriportData>(data: UserData<T>[]):
           });
           // now that we have a all the chunks for one customer, process them
           const settings = await getSettingsOrFail({ id: cxId });
+
+          analytics({
+            distinctId: cxId,
+            event: EventTypes.query,
+            properties: {
+              method: "POST",
+              url: "/webhook/garmin",
+              apiType: ApiTypes.devices,
+            },
+          });
```

Comment:
I thought about passing in the request for method and URL but though just adding it would be cleaner

Reply:
I'd use the command name or business process being executed here, instead of the URL, because:
- this might be called by other places triggered by diff URLs;
- URLs are more related to access metrics/analytics, whereas this one seems to be a business process

If this is an access metrics, maybe it could live on the route?

Reply:
So its an access metric but because of the nature of the payload i wanted to put it here to be able to get all the cxId's

Reply:
Gotcha. NABD so we can release this sooner than later, but for future references, we could return these from the command.

Curious: is there a way to send a message to Posthog and update it afterwards with more data? This way we could send it right at the route start, and update it with IDs after the cmd responds.

Reply:
AFAIK we cant update events that have already been created